constrain/library/G36CoolingOnlyTerminalBoxCoolingAirflowSetpoint.py

```diff
@@ -33,25 +46,44 @@
 
 ### Data requirements
 
-- operation_mode: System operation mode
-- zone_state: Zone state (heating, cooling, or deadband (not in either heating or cooling))
-- v_cool_max: Maximum cooling airflow setpoint
-- v_min: Occupied zone minimum airflow setpoint
-- v_spt: Active airflow setpoint
+- mode_system: System operation mode
+  - Data Value Unit: enumeration
+  - Data Point Affiliation: System control
+
+- state_zone: Zone state
```

Comment:
we should list the potential enums here.

Reply:
Addressed